extensions/positron-python/src/client/interpreter/configuration/interpreterSelector/commands/setInterpreter.ts

```diff
@@ -719,3 +728,15 @@ function getGroup(item: IInterpreterQuickPickItem, workspacePath?: string) {
             return EnvGroups[item.interpreter.envType];
     }
 }
+
+// --- Start Positron ---
+/**
+ * Wraps the original filter function to include additional filtering logic.
+ * If no filter is provided, the returned function will only include the additional filtering logic.
+ * @param filter The original filter function
+ * @returns A new filter function that includes the original filter function and the additional filtering logic
+ */
+function filterWrapper(filter: ((i: PythonEnvironment) => boolean) | undefined) {
+    return (i: PythonEnvironment) => (filter ? filter(i) : true) && shouldIncludeInterpreter(i.path);
+}
+// --- End Positron ---

```

Comment:
@seeM
> the interpreters are still shown in the "Python: Select Interpreter" command. Is that expected?

Thank you for catching this! I checked the "include" case, but missed the "exclude" case for the quickpick. This should be fixed now!